docs/source/guide/guide-pec.rst

```diff
@@ -0,0 +1,8 @@
+.. pec:
+
+*********************************************
+Probabilistic Error Cancellation
+*********************************************
+
+Probabilistic error cancellation (PEC) capabilities are currently being added to Mitiq. The :py:mod:`pec` sub-package contains the relative modules. A minimal example of the use of PEC in Mitiq can be found in the :ref:`Getting Started section <pec_getting_started>` of the guide, while more details on the theory and limitation of this technique can be found in
+the :ref:`Probabilistic Error Cancellation <guide_qem_pec>` section reviewing quantum error mitigation techniques.
```

Comment:
```suggestion
the :ref:`About Error Mitigation <guide_qem_pec>` section.
```
Maybe renaming the link in this way (while keeping the link to `guide_qem_pec`) is less confusing for the reader.